### PR TITLE
NF3 Mousetrap Readme

### PR DESCRIPTION
**Pull request by CarltonHenderson**
# Problem Context

NF3 Readme was spartan

## Changes

Flesh it out but remove the story-telling aspects of the Hackster.io post.

### CODE REVIEW

**Comment by CarltonHenderson** [comment]
@Mechanochist Requesting your review here because you're fairly unfamiliar with my Moustrap project. So if you could read the readme and tell me if it's pretty understandable that'd be awesome. Everyone on my side of the company already pretty much knows how it works so I'm afraid it'll be harder for them to notice oversights.

**Comment by CarltonHenderson** [comment]
FYI, here's the best place to read the readme in this PR
https://github.com/blues/app-accelerators/tree/carl-nf3-readme/better-mouse-trap

File: better-mouse-trap/README.md [hunk 1]
```diff
@@ -2,6 +2,103 @@
 
 A smarter mouse trap with SMS alerting.
 
-![](../app-banners/nf3.png)
+Find the complete story in the detailed [Hackster.io writeup].
 
-You can find all instructions for building this project in [its Hackster writeup](https://www.hackster.io/hendersoncarlton/i-love-checking-on-mousetraps-said-no-one-ever-52c5e7). This repo contains the project’s [JSONata configuration](./mousetrap.jsonata) for reference.
\ No newline at end of file
+[![Mouse wearing a helmet grabbing cheese from a trap](../app-banners/nf3.png)][hackster.io writeup]
+
```
Comment (tjvantoll) [inline better-mouse-trap/README.md:8]: As a suggestion I like having a mini TOC at the top of these guides. For example see the "To run this project yourself” sections at the top of https://github.com/blues/app-accelerators/tree/main/indoor-floor-level-tracker and https://github.com/blues/app-accelerators/tree/main/fridge-fleet-monitor.

File: better-mouse-trap/README.md [hunk 2]
```diff
@@ -2,6 +2,103 @@
...
+gateway, thanks to the onboard Notecard, forwards the message through Wi-Fi to
+my home router.
+
+![View inside the trap showing underside of lid](<readme-images/nf3_mousetrap_(8)_gUpTQaXMpg.avif>)
+![Sparrow Gateway wiring and logical data path from Lora Radio Signal to Sparrow essentials board, to qwiic i2c cable, to notecarrier, to notecard, to wifi](<readme-images/nf3_mousetrap_(13)_g4ofhDRGNm.avif>)
+
+### Cloud Setup
+
+- Sign up for a free account on [Notehub.io], and create a new project.
+- Configure the Notecard on the Sparrow Gateway with your Notehub ProductID
```
Comment (tjvantoll) [inline better-mouse-trap/README.md:66]: ProductUID

File: better-mouse-trap/README.md [hunk 3]
```diff
@@ -2,6 +2,103 @@
...
+
+![Notehub.io event screenshot](readme-images/image_X1HU4mO9mN.avif)
+
+![Route to Twilio SMS Service](<readme-images/nf3_mousetrap_(19)_TOXG7lKQRu.avif>)
+
+![Phone Screenshot of SMS](readme-images/image_S2jSMIx7G8.avif)
+
+### Final Thoughts
+
+In just a few hours, I was able to make a connected mousetrap solution that communicates anywhere on my property using LoRa thanks to [Sparrow]. Thanks to LoRa’s excellent signal propagation, traps can even be put in my detached garage, made of concrete block, where Wi-Fi doesn’t reach. As a bonus, if I ever want to move my mousetraps to a property where I don’t have internet access (like a storage location on the other side of town, or a grain silo on the other side of the country) I can easily swap in a cellular [Notecard] (with 10 years of data included) for about $60 (as of 2021).
```
Comment (tjvantoll) [inline better-mouse-trap/README.md:89]: Maybe kill this section as it feels more story-like, and different from the tone of the rest of the README.

File: better-mouse-trap/README.md [hunk 4]
```diff
@@ -2,6 +2,103 @@
...
+
+### Blues Wireless Community
+
+We’d love to hear bout you and your project on the [Blues Wireless Community Forum].
```
Comment (tjvantoll) [inline better-mouse-trap/README.md:93]: about

File: better-mouse-trap/mousetrap.jsonata [hunk 5]
```diff
@@ -1,20 +1,36 @@
+/* Declare a variable scope */
 (
-    /* User Config */
+    /************** User Config **************/
+
     $TrapLocations := {
-        "2037335832365003002c0047": "Location1",
-        "203733583236500375754859": "Location2"
+        "2037335832365003002c0047": "Garage",
+        "203733583236500375754859": "Basement"
```
Comment (tjvantoll) [inline better-mouse-trap/mousetrap.jsonata:7]: oooh I like the User Config comments

File: better-mouse-trap/README.md [hunk 6]
```diff
@@ -2,6 +2,103 @@
...
 A smarter mouse trap with SMS alerting.
 
-![](../app-banners/nf3.png)
+Find the complete story in the detailed [Hackster.io writeup].
 
-You can find all instructions for building this project in [its Hackster writeup](https://www.hackster.io/hendersoncarlton/i-love-checking-on-mousetraps-said-no-one-ever-52c5e7). This repo contains the project’s [JSONata configuration](./mousetrap.jsonata) for reference.
\ No newline at end of file
+[![Mouse wearing a helmet grabbing cheese from a trap](../app-banners/nf3.png)][hackster.io writeup]
+
+## TL;DR Instructions
```
Comment (tjvantoll) [inline better-mouse-trap/README.md:9]: Is this heading necessary?

I feel like you already imply that this is the quick-and-dirty version by linking to Hackster for the “complete” version earlier.

**Review by tjvantoll — APPROVED** [review]
I noted a few trivial things but nothing too important. Looks good!

**Comment by Mechanochist** [comment]
These instructions were quite easy for me to follow, and were clear and concise.  I would have no trouble reproducing this quickly based on the instructions!  One thing to note is that Safari does not render .avif images, so most of the images wouldn't load until I switched to Chrome.

**Comment by CarltonHenderson** [comment]
Awesome feedback thanks!
________________________________

**Pull request merged**